radio_beam/utils.py

```diff
@@ -16,6 +19,99 @@ class RadioBeamDeprecationWarning(Warning):
     pass
 
 
+def deconvolve_opt(beamprops1, beamprops2, failure_returns_pointlike=False):
```

Comment:
why not `optimized_deconvolve` or `deconvolve_optimized`?

Reply:
Switched to `deconvolve_optimized`